hackathon/views.py

```diff
@@ -1,10 +1,61 @@
 from django.views.generic import ListView
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from datetime import datetime
 
-from .models import Hackathon
+from .models import Hackathon, HackAwardCategory, HackTeam, HackProject, HackProjectScore, HackProjectScoreCategory
+from .forms import HackathonForm
 
 
 class HackathonListView(ListView):
     """Renders a page with a list of Hackathons."""
     model = Hackathon
     ordering = ['-created']
     paginate_by = 8
+
+
+@login_required
+def create_event(request):
```

Comment:
I would keep the naming convention consistent `create_hackathon`

Reply:
Same applies for any instances in the project please

Reply:
This has been done in line with one of the previous comments related to this issue.